main.tf

```diff
@@ -1,38 +1,189 @@
 locals {
-  name          = "my-module"
+  name          = "ibm-cpd-wkc-instance"
   bin_dir       = module.setup_clis.bin_dir
-  yaml_dir      = "${path.cwd}/.tmp/${local.name}/chart/${local.name}"
+  prerequisites_name = "ibm-cpd-wkc-prereqs"
+  prerequisites_yaml_dir = "${path.cwd}/.tmp/${local.name}/chart/${local.prerequisites_name}"
+  subscription_name = "ibm-cpd-wkc-subscription"
+  subscription_yaml_dir = "${path.cwd}/.tmp/${local.name}/chart/${local.subscription_name}"
+  instance_yaml_dir      = "${path.cwd}/.tmp/${local.name}/chart/${local.name}"
   service_url   = "http://${local.name}.${var.namespace}"
-  values_content = {
+
+  subscription_content = {
+    name = "ibm-cpd-wkc-operator-catalog-subscription"
+    operator_namespace = var.operator_namespace
+    syncwave = "-5"
+    channel = "v1.0"
+    installPlan = "Automatic"
+  }
+
+  instance_content = {
+    cpd_namespace = var.cpd_namespace
+    name = "wkc-cr"
+    version = "4.0.5"
+    license = "Enterprise"
+    storageVendor = "portworx"
+    wkc_set_kernel_params = "True"
```

Comment:
These should be boolean values - `true` without the quotes

Reply:
This is required as a string values as per documentation : https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=catalog-installing-watson-knowledge-service